cmd/fiovb.c

```diff
@@ -142,11 +142,35 @@ int do_fiovb_delete_pvalue(cmd_tbl_t *cmdtp, int flag, int argc,
 	return CMD_RET_FAILURE;
 }
 
+int do_is_secondary_boot(cmd_tbl_t *cmdtp, int flag, int argc,
+			 char * const argv[])
+{
+	bool secondary;
+
+	if (!fiovb_ops) {
+		printf("Foundries.IO Verified Boot is not initialized, run 'fiovb init' first\n");
+		return CMD_RET_FAILURE;
+	}
+
+	if (argc != 1)
+		return CMD_RET_USAGE;
+
+	if (fiovb_ops->is_secondary_boot(fiovb_ops, &secondary) ==
+	    FIOVB_IO_RESULT_OK) {
+		printf("Secondary bit = %d\n", secondary);
```

Comment:
it made me think if we need a more verbose return here. I like the other printf line (with the run 'fiovb init' first), and I wonder what happen if the result of this command is only "Secondary bit 1"

anyway, only aesthetic comment, if this is the only one, I would say go for the merge.

Reply:
I wonder if it is better to just not output anything, as I get the impression we're currently way too verbose with fiovb.

This will later be used as part of our boot script, right?

Reply:
@ricardosalveti exactly.
Return code is enough to this cmd in `if else` build-it statements in U-Boot shell
However additional output would be good for debugging purposes (so I can just `s/printf/debug/g` here)